src/javascript/app/common/guide.js

```diff
@@ -43,12 +44,8 @@ const Guide = (() => {
             $(opt.guideBtnID).remove();
             return;
         }
-
         makeButton();
     };
-    /*
```

Comment:
Any reason we are removing this comment? Seems useful and consistent with other functions. Maybe we can discard changes on this file if it's not intentional.

Reply:
Returned.